src/runtime.js

```diff
@@ -344,7 +344,7 @@ function adoptShared(global, callback) {
  * @return {!Promise}
```

Comment:
Focus on this file? Since lightbox gallery was the only user of this function, I went ahead and deleted it. Is there good reason to keep it alive?

Reply:
:+1: